en/content-workflow/camera-config.md

```diff
@@ -0,0 +1,18 @@
+# Scene camera configuration panel
+
+The scene camera configuration panel can configure the Camera operation and render properties in the **Scene**.
```

Comment:
the operation and rendering of the built-in camera used to render the scene editor.